### DISCUSSION TITLE
`rich|advanced_text_box` support (HTML, Markdown, etc...)

### DISCUSSION THREAD

**Discussion by qicosmos** [cycfi/elements#342] [Elements]
If show a markdown document with textedit, elements can't render markdown now, so do you have the plan to support it?

**Comment by djowel** [comment]
No, there's no plan to support markdown in textedit. If you want to work on it, I'll consider a PR. Feel free to discus in the Discored forum: [discord channel](https://discord.gg/NJkadSx)
